src/PlayAudioUsingMediaStreams/Controllers/AnimalSoundboardController.cs

```diff
@@ -1,26 +1,33 @@
 using System.Net.WebSockets;
 using System.Text;
 using System.Text.Json;
+using Google.Api.Gax.Grpc;
+using Google.Cloud.Speech.V1;
 using Microsoft.AspNetCore.Mvc;
 using PlayAudioUsingMediaStreams.WebApi.Services;
-using Twilio.AspNet.Core;
-using JsonSerializer = System.Text.Json.JsonSerializer;
 
 namespace PlayAudioUsingMediaStreams.WebApi.Controllers;
- 
+
 [ApiController]
 [Route("[controller]")]
-public class AnimalSoundboardController : TwilioController
+public class AnimalSoundboardController : Controller
 {
     private readonly SoundService _soundService;
     private readonly SpeechRecognitionService _speechRecognitionService;
+    private readonly IHostApplicationLifetime _applicationLifetime;
 
-    public AnimalSoundboardController(SoundService soundService, SpeechRecognitionService speechRecognitionService)
+    public AnimalSoundboardController(
+        SoundService soundService,
+        SpeechRecognitionService speechRecognitionService,
+        IHostApplicationLifetime applicationLifetime
```

Comment:
When the .NET app is stopped, it should close the websocket gracefully, or Twilio will keep the call open. Unfortunately, Twilio doesn't end the call when the websocket is aborted.
This may be unnecessary for the sample app.